WordPress/src/main/java/org/wordpress/android/ui/mysite/MySiteViewModel.kt

```diff
@@ -354,13 +354,17 @@ class MySiteViewModel @Inject constructor(
                     state.dashboardCardsAndItems.filterIsInstance<DashboardCards>().firstOrNull()
                 )
 
+                dashboardCardDomainUtils.trackDashboardCardDomainShown(viewModelScope, state as? SiteSelected)
```

Comment:
- **Move code:** Can we move `dashboardCardDomainUtils.trackDashboardCardDomainShown(viewModelScope, state as? SiteSelected)` in `trackCardsAndItemsShownIfNeeded(state)`?